### PR TITLE
Awesome books es6

### PR DESCRIPTION
**Pull request by karam084**
*- Project requirements

- Create a new directory and initialize it with git.
- Use npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.
- Re-create your Awesome books app following the rules:
- Use ES6 syntax.
- Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
- Keep all funcionalities of the app without errors.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+  }
+};
+
+const Remove = () => {
+  if (this.id > -1) {
+    bList.RemoveBook(this.id);
+    localStorage.setItem('BookList', JSON.stringify(bList.Books));
+    window.location.reload();
+  }
+};
```
Comment (er-contreras) [inline index.js]: - [x] It's not possible to add and remove a book. It through undefined.

Screen Shot 2022-02-21 at 16 15 13

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @karam084, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (rloterh) [reply index.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Karam, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂


<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**